src/Scaffold_Command.php

```diff
@@ -372,8 +372,7 @@ function child_theme( $args, $assoc_args ) {
 			'theme_uri'  => ""
 		) );
 		$data['slug'] = $theme_slug;
-		$data['parent_theme_function_safe'] = str_replace( '-', '_', $data['parent_theme'] );
-
+		$data['parent_theme_function_safe'] = str_replace( array( ' ', '-' ), '_', $data['parent_theme'] );
```

Comment:
@miya0001 To clarify further, I think this pull request is fine because it fixes the immediate issue. It's also a _more specific_ change than using `preg_replace( '/\W/'`, which could have unintended consequences.